array-prototype-find.js

```diff
@@ -1,9 +1,7 @@
 // Array.prototype.find - MIT License (c) 2013 Paul Miller <http://paulmillr.com>
 // For all details and docs: https://github.com/paulmillr/array.prototype.find
 (function(globals){
-  if (Array.prototype.find) return;
-
-  var find = function(predicate) {
+  Array.prototype.find = function find (predicate) {
```

Comment:
Hmm, I'm wanting to get the shimming behaviour without the detect. What do you think about having three files? `detect.js`, `polyfill.js`, `shim.js`?